framework/widgets/LinkPager.php

```diff
@@ -218,7 +224,7 @@ protected function renderPageButton($label, $page, $class, $disabled, $active)
         if ($disabled) {
             Html::addCssClass($options, $this->disabledPageCssClass);
 
-            return Html::tag('li', Html::tag('span', $label), $options);
+            return Html::tag('li', Html::tag('span', $label, $this->disabledSpanOptions), $options);
```

Comment:
It makes sense to make tag name customizable as it's done [here](https://github.com/yiisoft/yii2/blob/master/framework/helpers/BaseHtml.php#L1225), for example